tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -712,3 +712,28 @@ Feature: Sharing files and folders with internal users
       | shared-resource |
       | lorem.txt       |
       | simple-folder   |
+
+  Scenario: change existing expiration date of an existing share after the maximum expiry date is enforced
+    Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
+    And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "7"
+    And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
+    And user "user1" has created a new share with following settings
+      | path      | lorem.txt |
+      | shareWith | user2     |
+    And user "user1" has logged in using the webUI
+    When the user tries to edit the collaborator expiry date of "User Two" of resource "lorem.txt" to "+14" days using the webUI
+    Then a "required" label should be visible above the expiration date field
+    And the expiration date for "+14 days" should be disabled
+    Then the expiration date shown on the webUI should be "+7" days
+    And it should not be possible to save the pending share on the webUI
+
+  Scenario: create a share without expiration date and then setting default expiration date and enforcing it does not change the expiration of previously created share
```

Comment:
```suggestion
  Scenario: setting default expiration date and enforcing it does not change the expiration-date of a previously created share
```

Would be good to have a similar test where the expiration date was set to something beyond the new maximum date, before the enforcement

Reply:
fixed